code/modules/mob/living/living_say.dm

```diff
@@ -142,9 +142,6 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 			say_dead(original_message)
 			return
 
-	if(HAS_TRAIT(src, TRAIT_SOFTSPOKEN) && !HAS_TRAIT(src, TRAIT_SIGN_LANG)) // softspoken trait only applies to spoken languages
```

Comment:
should probably comment this out instead of deleting it

Reply:
smh you and your perfectly justified complaints, anyway done